.travis.yml

```diff
@@ -104,20 +104,12 @@ env:
     - TRAVIS_FLAVOR=win32_event_log
     # END OF TRAVIS MATRIX
 
-before_install:
-  - sudo apt-get update ; sudo apt-get install -y curl apt-transport-https git netcat net-tools libpq-dev
-
 install:
   - bundle install
   - bundle package
   - git -C $HOME/dd-agent pull || git clone -b $DD_AGENT_BRANCH --depth 1 https://github.com/DataDog/dd-agent.git $HOME/dd-agent
   - git -C $HOME/integrations-extras pull || git clone -b $EXTRAS_BRANCH --depth 1 https://github.com/DataDog/integrations-extras.git $HOME/integrations-extras
   - echo "$HOME/dd-agent/" > $(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")/datadog-agent.pth
-  - pip install 'docker-compose==1.9.0'
```

Comment:
I added it to requirements-test.txt, I think docker is always supposed to run after `install` ?